src/components/DataLayoutWrapper.js

```diff
@@ -122,7 +122,7 @@ class DataLayoutWrapper extends Component {
 DataLayoutWrapper.propTypes = {
   onRowEdited: PropTypes.func,
   windowType: PropTypes.string,
-  viewId: PropTypes.number,
+  viewId: PropTypes.string,
```

Comment:
I noticed that the ids are either coming as a string or as a number. Normally all should be numbers right? I also put .any as that was annoying. I suspect some issue on top level components when these are passed. I guess we leave it like this and when we bump into the issue again we should fix it at the top level component sending those.